assignments/objects.js

```diff
@@ -18,27 +18,71 @@ let example = {
 }
 
 // Write your intern objects here:
+let internOne = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy@psu.edu",
+  "gender": "F"
+}
 
+let internTwo = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": function() {
+    console.log("Hello, my name is Kennan!");
```

Comment:
Consider using this.name instead of Keenan inside the console.log to see a very interesting result. Like so:
```console.log("Hello, my name is " + this.name + "!");```
Then you can use the same pattern to approach all other objects in the future, the keyword `this` refers to the object in which the property is contained. in this case this refers to internTwo.